packages/ui/src/exports/elements.ts

```diff
@@ -48,7 +48,11 @@ export { UnpublishMany } from '../elements/UnpublishMany/index.js'
 export { Upload } from '../elements/Upload/index.js'
 export { BlocksDrawer } from '../fields/Blocks/BlocksDrawer/index.js'
 import * as facelessUIImport from '@faceless-ui/modal'
+export { SetViewActions } from '../providers/Actions/SetViewActions/index.js'
 const { Modal } =
   facelessUIImport && 'Modal' in facelessUIImport ? facelessUIImport : { Modal: undefined }
 export { Modal }
-export { SetViewActions } from '../providers/Actions/SetViewActions/index.js'
+import * as reactToastifyImport from 'react-toastify'
+const { toast } =
+  reactToastifyImport && 'toast' in reactToastifyImport ? reactToastifyImport : { toast: undefined }
```

Comment:
Is this funky import necessary? I'm assuming so but its not pretty.

Reply:
I followed the example for the faceless UI, the only other third party export